inc/ThirdParty/Hostings/WPEngine.php

```diff
@@ -0,0 +1,155 @@
+<?php
+
+namespace WP_Rocket\ThirdParty\Hostings;
+
+use WP_Rocket\ThirdParty\ReturnTypesTrait;
+use WP_Rocket\Event_Management\Subscriber_Interface;
+use WpeCommon;
+
+/**
+ * Compatibility class for WP Engine.
+ *
+ * @since 3.6.1
+ */
+class WPEngine implements Subscriber_Interface {
+	use ReturnTypesTrait;
+
+	/**
+	 * Array of events this subscriber wants to listen to.
+	 *
+	 * @since 3.6.1
+	 *
+	 * @return array
+	 */
+	public static function get_subscribed_events() {
+		if (
+			! (
+				class_exists( 'WpeCommon' )
+				&&
+				function_exists( 'wpe_param' )
+			)
+		) {
+			return [];
+		}
+
+		return [
+			'rocket_varnish_field_settings'           => 'varnish_addon_title',
+			'rocket_display_input_varnish_auto_purge' => 'return_false',
+			'rocket_cache_mandatory_cookies'          => [ 'return_empty_array', PHP_INT_MAX ],
+			'rocket_advanced_cache_file'              => 'return_empty_string',
+			'admin_init'                              => [
+				[ 'remove_notices' ],
+				[ 'run_rocket_bot_after_wpengine' ],
+			],
+			'set_rocket_wp_cache_define'              => 'return_true',
+			'do_rocket_generate_caching_files'        => 'return_false',
+			'after_rocket_clean_domain'               => 'clean_wpengine',
+			'rocket_buffer'                           => [ 'add_footprint', 50 ],
+			'rocket_disable_htaccess'                 => 'disable_htaccess',
```

Comment:
@engahmeds3ed What if here, we return `true`?

```suggestion
			'rocket_disable_htaccess'                 => 'return_true',
```

We want to disable htaccess for customers on WPEngine.

Reply:
Yes
I was about to do that.
we was using the function to check on the php version but now we removed this check so I will return true to disable dealing with htaccess file forever on wpengine

Reply:
@arunbasillal Solved and checked on wpengine account.
Could u plz give it a look on wpengine?